docs/demo.md

```diff
@@ -0,0 +1,33 @@
+# fn demo
+
+This command will help you to create a kind cluster to install OpenFunction and its demo.
+
+## Parameters
+
+```shell
+--dry-run          Used to prompt for the components and their versions to be installed by the current command.
+--region-cn        For users in China to speed up the download process of dependent components.
+--delete           Remove the demo environment
+--verbose          Show verbose information.
+--timeout duration   Set timeout time. Default is 5 minutes. (default 5m0s)
+```
+
+## Use Cases
+
+#### Remove the demo environment
+
+```shell
+fn demo --delete
+```
+
+
+
+#### Support for users in China to speed up the installation process
```

Comment:
Support for users in China to speed up the installation process
=>
Install OpenFunction with limited access to gcr.io or github.com